tests/trello/gui/resources/components/ShownMenu.resource

```diff
@@ -0,0 +1,46 @@
+*** Settings ***
+Documentation    Variables and Keywords for Trello's Board Page Menu that is Show when clicking
+...              show menu in the top bar 
+...              (default page after logging in)
```

Comment:
If I'm not mistaken, this menu can only be seen from inside a board, so it's not the default page after logging in...